knowledge/faq.rst

```diff
@@ -35,7 +35,7 @@ that you get an error like the following one:
     shared=False
 
     ERROR: Missing prebuilt package for 'zlib/1.2.11'. You can try:
-        - List all available packages using 'conan list {ref}:* -r=remote'
+        - List all available packages using 'conan list "{ref}:*" -r=remote'
```

Comment:
Probably this deserves a PR to Conan client too

Reply:
You mean the help output of `conan list --help`?
```
positional arguments:
  pattern               A pattern in the form 'pkg/version#revision:package_id#revision', e.g: zlib/1.2.13:* means all binaries for zlib/1.2.13. If revision is not specified, it is assumed latest one.
```

I'll update it :)

Reply:
Nope, the error message that produces ``ERROR: Missing prebuilt package for 'zlib/1.2.11'``.

Reply:
Okey, well in that case both lines should be changed, the list --help and that error. I'll change it

Reply:
Already fixed and merged on conan client